Snowflake.Data/Configuration/EasyLoggingConfigParser.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Snowflake.Data.Configuration
+{
+    internal class EasyLoggingConfigParser
+    {
+        public virtual EasyLoggingConfig Parse(string filePath)
+        {
+            var configFile = TryToReadFile(filePath);
+            return configFile == null ? null : TryToParseFile(configFile);
+        }
+
+        private string TryToReadFile(string filePath)
+        {
+            if (string.IsNullOrEmpty(filePath))
+            {
+                return null;
+            }
+            try
+            {
+                return File.ReadAllText(filePath);
+            }
+            catch (Exception e)
+            {
+                Console.WriteLine("Finding easy logging configuration failed");
```

Comment:
don't we want to use default logger then?

Reply:
Just an oversight. Changed.